File: ab/archivos_base/16. Metodos/Final/JavaScript/js/Semaforo.js
function Semaforo(numLuces, luzActiva){
	this.numLuces = numLuces;
	this.luzActiva = luzActiva;	
}

Semaforo.prototype.cambiarLuz = function(nuevaLuz){
	this.luzActiva = nuevaLuz;
}

File: ab/archivos_base/16. Metodos/Final/Swift/Saludo/Saludo/Semaforo.swift
//
//  Semaforo.swift
//  Saludo
//
//  Created by trainermac on 26/11/15.
//  Copyright © 2015 trainermac. All rights reserved.
//

import Foundation

class Semaforo:NSObject{
    var numLuces:Int = 0
    var luzActiva:String = ""
    
    func cambiarLuz(nuevaLuz:String){
        luzActiva = nuevaLuz
    }
}